generics/constants/api-responses.js

```diff
@@ -286,6 +286,7 @@ module.exports = {
 	ADMIN_TOKEN_MISSING_CODE: 'ERR_ADMIN_TOKEN_MISSING',
 	ADMIN_TOKEN_MISSING_MESSAGE: 'Admin token is missing',
 	SOLUTION_NOT_ALLOWED_TO_BE_CONSUMED: 'This solution is not allowed to be consumed privately',
+	COMPONENT_ORDER_DUPLICATE: 'Component order cannot be duplicate',
```

Comment:
@borkarsaish65 COMPONENT_ORDER_DUPLICATE_FOUND - update the variable name

Reply:
done